code/modules/projectiles/guns/projectile/boltgun/baroque.dm

```diff
@@ -31,7 +31,7 @@ Own a musket for department defense, since that's what the founding hunters inte
 	fire_sound = 'sound/weapons/guns/fire/sniper_fire.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/rifle_load.ogg'
 	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_BONE = 20)
-	price_tag = 750
+	price_tag = 1750
```

Comment:
These one is a bit of a different thing, in the trade ships themselfs the price is set differently to try and make it valueable to sell in that case, so that might also need tweaking

Reply:
Possibly. Felt odd to leave it with such a low price despite it being intended to blow away lines of mobs. Would need some suggestions on its pricing, as it's incredibly good - not as good as an AMR but it's basically a mini-AMR